plugins/browser-plugin-youtube-tracking/src/helperFunctions.ts

```diff
@@ -28,40 +28,39 @@ export function trackingOptionsParser(mediaId: string, conf?: MediaTrackingOptio
   };
 
   if (!conf) return defaults;
+  if (conf.label) defaults.label = conf.label;
+  defaults.updateRate = conf.updateRate || defaults.updateRate;
+  defaults.captureEvents = conf.captureEvents || defaults.captureEvents;
 
-  if (conf.updateRate) defaults.updateRate = conf.updateRate;
-
-  if (conf.captureEvents) {
-    let parsedEvents: EventGroup = [];
-    for (let ev of conf.captureEvents) {
-      // If an event is an EventGroup, get the events from that group
-      if (EventGroups.hasOwnProperty(ev)) {
-        parsedEvents = parsedEvents.concat(EventGroups[ev]);
-      } else if (!Object.keys(AllEvents).filter((k) => k === ev)) {
-        console.warn(`'${ev}' is not a valid event.`);
-      } else {
-        parsedEvents.push(ev);
-      }
+  let parsedEvents: EventGroup = [];
+  for (let ev of defaults.captureEvents) {
+    // If an event is an EventGroup, get the events from that group
+    if (EventGroups.hasOwnProperty(ev)) {
+      parsedEvents = parsedEvents.concat(EventGroups[ev]);
+    } else if (!Object.keys(AllEvents).filter((k) => k === ev)) {
+      console.warn(`'${ev}' is not a valid event.`);
+    } else {
+      parsedEvents.push(ev);
     }
+  }
 
-    conf.captureEvents = parsedEvents;
+  defaults.captureEvents = parsedEvents;
 
-    for (let ev of conf.captureEvents) {
-      const youtubeEvent = CaptureEventToYouTubeEvent[ev];
-      if (CaptureEventToYouTubeEvent.hasOwnProperty(ev) && defaults.youtubeEvents.indexOf(youtubeEvent) === -1) {
-        defaults.youtubeEvents.push(youtubeEvent);
-      }
+  for (let ev of defaults.captureEvents) {
+    const youtubeEvent = CaptureEventToYouTubeEvent[ev];
+    if (CaptureEventToYouTubeEvent.hasOwnProperty(ev) && defaults.youtubeEvents.indexOf(youtubeEvent) === -1) {
```

Comment:
Accessing `CaptureEventToYouTubeEvent[ev]` and then checking `CaptureEventToYouTubeEvent.hasOwnProperty(ev)` seems a little odd. It'll be fine but it feels backwards. 

I'd be more tempted to just get rid of the `const` and access `CaptureEventToYouTubeEvent[ev]` twice.